pkg/relayer/proxy/sync.go

```diff
@@ -121,6 +141,7 @@ func (server *relayMinerHTTPServer) serveSyncRequest(
 		logger.Warn().Err(err).Msg("failed setting write deadline for response controller")
 		return relayRequest, ErrRelayerProxyInternalError.Wrap(err.Error())
 	}
+	instructionTimes.Record("set_response_controller_write_deadline")
```

Comment:
Optional: wdyt of moving all of the instructionTimes strings to an independent file so we know what's available rather than embedded magic strings?

Open to pushback